### PR TITLE
refactor: remove `AccountAssetsList`

### LINKED ISSUE #7080: [Task]: Remove `AccountAssetsList`

### PR DESCRIPTION
**Pull request by VmMad**
## Summary

Remove unnecessary component `AccountAssetsList`

## Changelog
Fixes #7080 
```
- Delete AccountAssetsList.svelte
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/wallet/Wallet.svelte [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
-    import { AccountAssetsList, Pane, ReceiveAddressButton } from '@ui'
+    import { AssetList, Pane, ReceiveAddressButton } from '@ui'
     import { AccountSummary, AccountActivity, SendButton } from '@components'
```
Comment (evavirseda) [inline packages/desktop/views/dashboard/wallet/Wallet.svelte:4]:
```suggestion
```

File: packages/desktop/views/dashboard/wallet/Wallet.svelte [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
+    import { selectedAccountAssets } from '@core/wallet'
     import { selectedAccount } from '@core/account/stores'
```
Comment (evavirseda) [inline packages/desktop/views/dashboard/wallet/Wallet.svelte:7]:
```suggestion
```

**Review by evavirseda — CHANGES_REQUESTED** [review]
there's a corner missing to sweep :broom: :wink:

**Review by evavirseda — APPROVED** [review]
:rocket:

**Review by begonaalvarezd — APPROVED** [review]
🚀

**Pull request merged**